en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,32 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
 
```

Comment:
Don't we need to include concepts defined in https://apim.docs.wso2.com/en/latest/get-started/key-concepts/ ?
However some of them like API Product, API Visibility, Workflows, Message Mediation Policies, Handler, Tenant, Service Catalog and roles might not be relevant to choreo as of now.

Reply:
Please let us know the list that applies to Choreo, and then we can proceed to add it here.

Reply:
Excluded list is mentioned above. So we can add the remaining items.

Reply:
Thanks. Will add the rest.